.github/workflows/render-smoke.yml

```
@@ -0,0 +1,114 @@
+name: Render Smoke Test
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)